akvo/rsr/spa/app/modules/program/ActualValue.jsx

```diff
@@ -1,43 +1,61 @@
-import React from 'react'
-import { Button } from 'antd'
+import React, { useState } from 'react'
+import { Button, message } from 'antd'
+import { connect } from 'react-redux'
+
 import Icon from '../../components/Icon'
 import Aggregation from './Aggregation'
 import {
-  actualValueIcons,
+  aggregatedIcons,
   callToAction,
-  jobStatus,
+  MAX_ATTEMPTS,
   toolTips
 } from './config'
+import api from '../../utils/api'
+import * as actions from './store/actions'
 
 const ActualValue = ({
+  updateJobStatus,
   actualValue,
   job = {},
 }) => {
-  const _status = (!job?.id && job?.status === jobStatus.maxxed) ? jobStatus.failed : job?.status
+  const [loading, setLoading] = useState(false)
+  const _status = job?.status
   const title = toolTips[_status] || null
-  const iconType = actualValueIcons[_status] || null
+  const iconType = aggregatedIcons[_status] || null
 
-  const handleOnRestartJob = () => {
-    console.log('call to API')
+  const handleOnRestartJob = (jobID) => {
+    setLoading(true)
+    api
+      .post(`/jobs/indicator_period_aggregation/${jobID}/reschedule/?format=json`)
+      .then(({ data }) => {
+        setLoading(false)
+        updateJobStatus(jobID, data)
+      })
+      .catch((err) => {
+        setLoading(false)
+        if (err) message.error('Failed to restart the job')
+      })
   }
 
   return (
     <Aggregation>
       {_status && (
         <Aggregation.Col icon>
-          <Aggregation.Tooltip title={title}>
-            {
-              (callToAction.includes(job?.status) && job?.id)
-                ? (
-                  <Button shape="circle" onClick={handleOnRestartJob}>
-                    <Icon type={iconType} width="16px" height="16px" className={job.status} />
+          {
+            (callToAction.includes(job?.status) && job?.id && job?.attempts < MAX_ATTEMPTS)
```

Comment:
Why not use only the job status here? :thinking:  Why also `MAX_ATTEMPTS`? @ifirmawan

Reply:
I thought that would be covered when the job nears the max attempt, but if it's not necessary then I will remove it